### PR TITLE
feat: Adds Airflow DAG for running some of the DIM checks

### PR DESCRIPTION
**Pull request by kik-kik**
# feat: Adds Airflow DAG for running some of the DIM checks

dim link: https://github.com/mozilla/dim

### CODE REVIEW

File: dags/data_monitoring.py [hunk 1]
```diff
@@ -6,40 +6,63 @@
...
+    schedule_interval="@daily",
+    doc_md=DOCS,
+    tags=TAGS,
+    catchup=True,
+) as dag:
+
+    for target_dataset in TARGET_DATASETS:
+        project_id, dataset, table = target_dataset.split(".")
+
+        task_name = f"rim_run_check_{target_dataset}"
```
Comment (alekhyamoz) [inline dags/data_monitoring.py:49]: nit: dim instead on rim

File: dags/data_monitoring.py [hunk 2]
```diff
@@ -6,40 +6,63 @@
...
+                f"--project_id={project_id}",
+                f"--dataset={dataset}",
+                f"--table={table}"
+                "--date_partition_parameter={{ ds }}"
+            ],
+            env_vars=dict(
+                SLACK_BOT_TOKEN="{{ var.value.dim_slack_secret_token }}"),
+            gcp_conn_id='google_cloud_gke_sandbox',
+            project_id='moz-fx-data-gke-sandbox',
+            cluster_name='akommasani-gke-sandbox',
```
Comment (alekhyamoz) [inline dags/data_monitoring.py:66]: shd this be changed to prod cluster?

**Review by alekhyamoz — APPROVED** [review]

File: dags/data_monitoring.py (hunk 1 above)
Comment (kik-kik) [reply dags/data_monitoring.py:49]: well spotted. Thanks!

File: bin/run [hunk 3]
```diff
@@ -95,6 +95,7 @@ init_connections() {
       "google_cloud_derived_datasets"
       "google_cloud_glam_fenix_dev"
       "google_cloud_shared_prod"
+      "google_cloud_gke_sandbox"
```
[the remark this answers is not in the document]
Comment (kik-kik) [reply bin/run:98]: I think this is used by local dev environments only anyways. So I think this should not be a problem.

File: dags/data_monitoring.py (hunk 2 above)
Comment (kik-kik) [reply dags/data_monitoring.py]: good point, I didn't notice this. Changed now.

**Pull request merged**